llvm/test/Transforms/LoopVectorize/simple_early_exit.ll

```diff
@@ -0,0 +1,1941 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
```

Comment:
This will need REQUIRES:asserts I think

Reply:
Ah yes, you're right! Good spot. I'll fix asap.